lib/update-check.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var latestVersion = require('latest-version');
+var semver = require('semver');
+var info = require('../package.json');
+var settings = require('../settings');
+var Spinner = require('cli-spinner').Spinner;
+var chalk = require('chalk');
+Spinner.setDefaultSpinnerString(Spinner.spinners[7]);
+
+function check(next) {
+	console.error('particle-cli v' + info.version);
+	console.error();
```

Comment:
By reducing the roundtrip you mean when talking on the forum or support tickets? Hopefully the number of people using old versions will go down with this feature!

I still think the version number will be additional noise for regular users. How about putting the current version in `displayVersionBanner` instead?